components/PlantCard.js

```diff
@@ -33,15 +40,26 @@ const CardContent = styled.div`
 `;
 
 const CardPlantName = styled.h3`
+  padding-left: 20px;
   margin-bottom: 0;
   color: white;
+  text-align: left;
 `;
 
 const CardBotanicalPlantName = styled.h4`
+  padding-left: 20px;
   margin-top: 5px;
   font-size: 14px;
   color: white;
   font-weight: 300;
+  text-align: left;
+`;
+
+const DetailsLink = styled.a`
```

Comment:
Instead of `a`, please use the Next.js’s `styled(Link)`  for client-side routing.
look here: [Link component](https://nextjs.org/docs/pages/api-reference/components/link)